test/components/RecordAttributes_test.js

```diff
@@ -226,5 +250,214 @@ describe("RecordAttributes component", () => {
+          expect(field.hasAttribute("disabled")).to.be.true;
+        });
+      });
+    });
+
+    describe("Schema version not in schema", () => {
+      let node;
+      describe("Schema version not in UISchema", () => {
+        beforeEach(() => {
+          node = createComponent(RecordAttributes, {
+            ...props,
+            collection,
+            record,
+          });
+        });
+
+        it("should not show the schema field", () => {
+          expect(node.querySelector("#root_schema")).to.not.exist;
+        });
+      });
+
+      describe("Schema version in UISchema", () => {
+        const withUISchema = {
+          ...collection,
+          record,
+          uiSchema: {
+            schema: {
+              "ui:widget": "textarea",
+            },
+          },
+        };
+
+        beforeEach(() => {
+          createComponent(RecordAttributes, {
+            ...props,
+            collection: withUISchema,
+          });
+        });
+
+        it("should not show the schema field", () => {
+          expect(node.querySelector("#root_schema")).to.not.exist;
+        });
+      });
+    });
+
+    describe("Last modified in schema", () => {
+      const record = {
+        data: {
+          id: "abc",
+          last_modified: 123,
+          foo: "bar",
+        },
+        permissions: {},
+      };
+
+      const withLastmodified = clone(collection);
+      withLastmodified.data.schema.properties.last_modified = {
+        type: "integer",
+      };
+
+      describe("Last modified not in UISchema", () => {
+        beforeEach(() => {
+          const node = createComponent(RecordAttributes, {
+            ...props,
+            collection: withLastmodified,
+            record,
+          });
+          field = node.querySelector("#root_last_modified");
+        });
+
+        it("should load the schema value", () => {
+          expect(field.value).to.eql("123");
+        });
+
+        it("should be a hidden field", () => {
+          expect(field.getAttribute("type")).to.eql("hidden");
+        });
+      });
+
+      describe("Last modified in UISchema", () => {
+        const withUISchema = clone(withLastmodified);
+        withUISchema.data.uiSchema = {
+          last_modified: {
+            "ui:widget": "text",
+          },
+        };
+
+        beforeEach(() => {
+          const node = createComponent(RecordAttributes, {
+            ...props,
+            collection: withUISchema,
+            record,
+          });
+          field = node.querySelector("#root_last_modified");
+        });
+
+        it("should load the schema value", () => {
+          expect(field.value).to.eql("123");
+        });
+
+        it("should show a custom field", () => {
+          expect(field.getAttribute("type")).to.eql("text");
+        });
+
+        it("should show the field as disabled", () => {
+          expect(field.hasAttribute("disabled")).to.be.true;
+        });
```

Comment:
I guess I don't really understand these tests. Should these fields be hidden *only* if they're not in the UI schema? Should we be checking that they are/are not hidden in the case where they are in the UI schema? Similarly, should they be disabled *only* if they're in the UI schema? Should we be checking that they are/are not disabled in the case where they aren't in the UI schema?